br.com.barros.DevWidgets.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=DevWidgets
+Version=1.0
+Comment=Utilities for developers. 
+Exec=devwidgets %u
+Icon=icon.svg
```

Comment:
this is still incorrect. Name never include the extension.

See how you install the icon for the name.